src/addrman.cpp

```diff
@@ -748,15 +749,16 @@ CAddrInfo CAddrMan::Select_(bool newOnly) const
     }
 }
 
-int CAddrMan::Check_() const
+int CAddrMan::CheckAddrMan_() const
 {
     AssertLockHeld(cs);
 
     // Run consistency checks 1 in m_consistency_check_ratio times if enabled
     if (m_consistency_check_ratio == 0) return 0;
     if (insecure_rand.randrange(m_consistency_check_ratio) >= 1) return 0;
 
-    LogPrint(BCLog::ADDRMAN, "Addrman checks started: new %i, tried %i, total %u\n", nNew, nTried, vRandom.size());
+    LOG_TIME_MILLIS_WITH_CATEGORY(
+        strprintf("new %i, tried %i, total %u", nNew, nTried, vRandom.size()), BCLog::ADDRMAN);
```

Comment:
Looks like this will print the statistics twice, implying they would change in-between, which might be confusing.

Not sure how to best solve this.

Reply:
> Looks like this will print the statistics twice, implying they would change in-between, which might be confusing.
> 
> Not sure how to best solve this.

I'll have a look.

Reply:
Done, example on signet:
```
2021-09-11T19:39:05Z [scheduler] Enter: lock contention ::cs_main, validation.cpp:4874 started
2021-09-11T19:39:05Z [opencon] CheckAddrMan_: new 442, tried 57, total 499 started
2021-09-11T19:39:05Z [opencon] CheckAddrMan_: completed (10.30ms)
2021-09-11T19:39:07Z [scheduler] Enter: lock contention ::cs_main, validation.cpp:4874 completed (2906703μs)
```